web/src/components/task/task-sidebar/finish-button/finish-button.tsx

```diff
@@ -86,7 +86,9 @@ export const FinishButton: FC<TaskSidebarProps> = ({
     const showConfirmModal = () => {
         if (confirmWindowNeed === 'true') {
             uuiModals
-                .show<string>((props) => <ConfirmModal {...props} />)
+                .show<string>((props) => (
+                    <ConfirmModal modalProps={props} isValidating={isValidation} />
```

Comment:
What do you think to keep the same name for prop as for passed variable "isValidation"?

Reply:
Changed